<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnakeGame</class>
 <widget class="QDialog" name="SnakeGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snake Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #282828;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin" stdset="0">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_main">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="margin" stdset="0">
      <number>10</number>
     </property>
     <item>
      <widget class="QFrame" name="gameArea">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_infoPanel">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="margin" stdset="0">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="levelLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLabel {
               color: #ffffff;
           }
          </string>
         </property>
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="levelDisplay">
         <property name="styleSheet">
          <string notr="true">
           QLCDNumber {
               background-color: #000000;
               color: #00ff00;
               border: 2px solid #00ff00;
           }
          </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLabel {
               color: #ffffff;
           }
          </string>
         </property>
         <property name="text">
          <string>Score:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="scoreDisplay">
         <property name="styleSheet">
          <string notr="true">
           QLCDNumber {
               background-color: #000000;
               color: #00ff00;
               border: 2px solid #00ff00;
           }
          </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="highScoreLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
           QLabel {
               color: #ffffff;
           }
          </string>
         </property>
         <property name="text">
          <string>High Score:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="highScoreDisplay">
         <property name="styleSheet">
          <string notr="true">
           QLCDNumber {
               background-color: #000000;
               color: #00ff00;
               border: 2px solid #00ff00;
           }
          </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
background-color: #4CAF50;
color: white;
border: none;
border-radius: 10px;
padding: 10px;
}
QPushButton:hover {
background-color: #45a049;
}
QPushButton:pressed {
background-color: #388e3c;
}
          </string>
         </property>
         <property name="text">
          <string>Back to Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
